tests/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: Python CI
+
+on: [push, pull_request]
```

Comment:
should be changed to
on:
  pull_request:
  push:
    branches:
      - main